core/controllers/domain_objects_validator.py

```diff
@@ -35,4 +36,20 @@ def validate_exploration_change(obj):
     """
     # No explicit call to validate_dict method is necessary, because
     # ExplorationChange calls it while initialization.
+
     exp_domain.ExplorationChange(obj) # type: ignore[no-untyped-call]
+
+
+def validate_new_config_property_values(obj):
```

Comment:
Please do add backend tests for this validator.

Reply:
Done